solr/contrib/ltr/src/java/org/apache/solr/ltr/model/MultipleAdditiveTreesModel.java

```diff
@@ -170,19 +170,21 @@ public boolean isLeaf() {
     }
 
     public float score(float[] featureVector) {
-      if (isLeaf()) {
-        return value;
-      }
-
-      // unsupported feature (tree is looking for a feature that does not exist)
-      if  ((featureIndex < 0) || (featureIndex >= featureVector.length)) {
-        return 0f;
-      }
+      RegressionTreeNode regressionTreeNode = this;
```

Comment:
RegressionTreeNode regressionTreeNode 
I am usually not a big fan of type = variable name approach, not much information is added with the name
maybe a simple "current" ?